capture/parsers/http.c

```diff
@@ -332,6 +340,28 @@ LOCAL int moloch_hp_cb_on_header_value (http_parser *parser, const char *at, siz
     LOG("HTTPDEBUG: which: %d value: %.*s", http->which, (int)length, at);
 #endif
 
+/*------------VISA Enhancement Begin---------------*/
+    if(http->which == 0) { // Headers in HTTP request
+        if (config.parseHTTPHeaderRequestAll) {
+            char *req_header_field = g_strdup(http->header[http->which]);
```

Comment:
There is no reason to do a strdup, and then a copy, and then a free.  
            moloch_field_string_add(headerReqField, session, http->header[http->which], -1, TRUE);
            moloch_field_string_add(headerReqValue, session, at, length, TRUE);

same for the below.  BTW, if the value is across calls this will not work since you should be concating the values like the inValue stuff does.